js/modules/loki_message_api.js

```diff
@@ -211,17 +209,19 @@ class LokiMessageAPI {
 
         // Make sure we aren't doing too much PoW
         const currentDifficulty = window.storage.get('PoWDifficulty', null);
-        const newDifficulty = result.difficulty;
-        if (newDifficulty != null && newDifficulty !== currentDifficulty) {
-          window.storage.put('PoWDifficulty', newDifficulty);
-        }
+        if (result && result.difficulty) {
```

Comment:
```suggestion
        if (result && result.difficulty && result.difficulty !== currentDifficulty) {
```
We can move the lower if statement to the one above